python/ql/src/experimental/semmle/python/frameworks/Stdlib.qll

```diff
@@ -98,3 +99,47 @@ private module Re {
     override DataFlow::Node getRegexNode() { result = regexNode }
   }
 }
+
+// ---------------------------------------------------------------------------
+// xmltodict
+// ---------------------------------------------------------------------------
+/** Gets a reference to the `xmltodict` module. */
+API::Node xmltodict() { result = API::moduleImport("xmltodict") }
+
+/**
+ * A call to `xmltodict.parse`
+ * See https://github.com/martinblech/xmltodict/blob/ae19c452ca000bf243bfc16274c060bf3bf7cf51/xmltodict.py#L198
+ */
+private class XmlToDictParseCall extends Decoding::Range, DataFlow::CallCfgNode {
+  XmlToDictParseCall() { this = xmltodict().getMember("parse").getACall() }
+
+  override predicate mayExecuteInput() { none() }
+
+  override DataFlow::Node getAnInput() { result = this.getArg(0) }
+
+  override DataFlow::Node getOutput() { result = this }
+
+  override string getFormat() { result = "JSON" }
```

Comment:
```suggestion
  override string getFormat() { result = "XML" }
```

Would be great if this modeling of `xmltodict` could be moved to it's own file. But at least the format needs to be changed.

Reply:
I agree, and this change arises the last point you spot in the comment. `getFormat()` specifies the format it is being _converted from_, but I can't think of a way to know the format it will be _converted to_. Is there a way to know that?